Troupon.Catalog.Api/Startup.cs

```diff
@@ -36,19 +36,17 @@ public Startup(IConfiguration configuration)
 
     public void ConfigureServices(IServiceCollection services)
     {
-      services.AddOAuthGenericAuthentication(Configuration).AddOAuthM2MAuthFlow();
-
       services.AddControllers().AddNewtonsoftJson();
-      services.AddOAuthController();
-
-      services.AddAuthorization(options =>
-      {
-        options.AddPolicy(TenantPolicy.Key, pb => pb.AddTenantPolicy("pwc"));
-        options.AddPolicy(AdminOnlyPolicy.Key, pb => pb.AddAdminOnlyPolicy());
-      });
-
-      services.AddPolicyHandlers();
 
+      // AUTH IS PARKED
+      // services.AddOAuthGenericAuthentication(Configuration).AddOAuthM2MAuthFlow();
```

Comment:
For what I understood in my PRs reviews, we want to avoid having commented code (also a good practice). Is there an important reason for us to keep this code commented?

Reply:
No your right ! I should remove it.

I will add it into Wiki instead to remember how to add the Authentication and Authorization !

Thanks !